server/lib/ldap/Manager.ts

```diff
@@ -0,0 +1,403 @@
+import limax from 'limax';
+import { SHA256 } from 'meteor/sha';
+import { Meteor } from 'meteor/meteor';
+import { Accounts } from 'meteor/accounts-base';
```

Comment:
since this file is imported by a service, **ideally** it should not import any Meteor dependency. Doing so it won't work as a dedicated service.. so this is something we'll need to take a look in the future.